index.md

```diff
@@ -2,16 +2,77 @@
 layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
+date: '`r format(Sys.time(), "%d %B, %Y")`'
 ---
-FIXME: home page introduction
 
-<!-- this is an html comment -->
+This tutorial is built to teach you how to use ESMValTool.  
 
-{% comment %} This is a comment in Liquid {% endcomment %}
+The Earth System Model Evaluation Tool (ESMValTool) is a community developped software toolkit that aims to faciliate
+the diagnosis and evaluation of the causes and effects of model biases and inter-model spread within the CMIP model
+ensemble.
+
+
+> ## What will you learn in this course
+>
+>   - What is ESMValTool
+>
+>   - How to install ESMValTool
+>
+>   - How to run ESMValTool
+>
+>   - How to develop your own diagnostics and recipes
+>
+>   - How to contribute your recipes and diagnostics back into ESMValTool
+>
+{: .checklist}
 
 > ## Prerequisites
 >
-> FIXME
+> - Basic understanding of git (optional - but useful)
+>
+>  - Basic understanding of your preferred command line interface (ie a bash terminal)
+>
+>  - Access to CMIP data
+>
+>  - Access to a suitable computing system (eg Jasmin)
+>
+> - Github account (optional, but useful!)
```

Comment:
Single line breaks do not render in markdown. I would suggest to make a list or two separate lists, with optional and minimal requirements. Does the suitable computing system belong to the optional or minimal list? I would say you own laptop would also work for simple and small recipes and especially this tutorial should be doable on a home computer. Don't we need conda installed for this tutorial?

```suggestion
> *Minimal requirements:*
>   - Basic understanding of your preferred command line interface (i.e. a bash terminal)
>   - Access to CMIP data
>   - Laptop/desktop with [(mini)conda](https://docs.conda.io/en/latest/miniconda.html) installed
> 
> *Optional, but useful:*
>   - Access to a suitable computing system (eg Jasmin)
>   - Basic understanding of git
>   - Github account
```

Reply:
@ledm what do you think about this? My comment keeps disappearing because the text gets updated, but I still have the same questions with this list.

Reply:
I like this, sorry it keeps gettings pushed back. I'll try to get it in when I can.

Reply:
Added this in now